install/eclipse.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Installer for JetBrains Toolbox
+# This will allow the installation, upgrade and management of other JetBrains tools
+# Running the tool itself will automatically add an icon to the launcher
+
```

Comment:
We tend towards `unofficial bash strict mode` (http://redsymbol.net/articles/unofficial-bash-strict-mode/ - although this should be documented somewhere, will investigate) Could this use `set -euo pipefail` please?